html/user/per_app_list.php

```diff
@@ -63,13 +67,20 @@ function show_header($is_team, $apps, $appid, $is_total) {
 // show a user or team, with their credit for each app
 //
 function show_row($item, $apps, $is_team, $i) {
```

Comment:
I understand. But then the list will not be sorted. The sorting is done in the query that gets cached but the actual value is taken directly from the DB. This will lead to users that have a higher value listed after users with a lower value. This is perceived as a bug by the user. So I think we should cache everything right away in `get_top_items()`.